pkg/apis/batch/types.go

```diff
@@ -372,6 +399,15 @@ type JobStatus struct {
 	// +optional
 	Active int32
 
+	// The number of pods which are terminating (in phase Pending or Running
+	// and have a non-nil deletionTimestamp).
+	//
+	// This field is alpha-level. The job controller populates the field when
+	// the feature gate JobPodReplacementPolicy is enabled (disabled by default).
+	// to be turned on.
```

Comment:
```suggestion
```

Reply:
still pending

Reply:
You got me as I was waiting for the API to generate.  Updated now.